source/javascripts/controllers/_ProvProfileController.js.erb

```diff
@@ -232,8 +232,6 @@ angular.module("BitriseWorkflowEditor").controller("ProvProfileController", func
 		}
 		else if (_.any(provProfile.developerCertificateIdentities, function(anIdentity) {
 			return !isIdentityWithSerialAddedToApp(anIdentity.serial);
-		}) || _.any(provProfile.provisionedDevices, function(aDevice) {
```

Comment:
This means, prov profile will have red - missing - status if any certs or any devices are missing. Shouldn't it be "all certs missing" instead of "any certs missing"? What about the devices? One device is enough for success status, or all is required?